ts/store/reducers/persistedPreferences.ts

```diff
@@ -57,3 +68,8 @@ export default function preferencesReducer(
 
   return state;
 }
+
+export const isPagoPATestEnabledSelector = (state: GlobalState) =>
+  state.persistedPreferences.isPagoPATestEnabled === true
```

Comment:
```suggestion
export const isPagoPATestEnabledSelector = (state: GlobalState) => state.persistedPreferences.isPagoPATestEnabled
```

Reply:
@gunzip actually the comparison is needed because `isPagoPATestEnabled` will be `undefined` initially

Reply:
@cloudify I agree with @gunzip, it can't be undefined because it is set false in the initialPreferencesState. The problem is that it must not be optional in the type PersistedPreferencesState.

Reply:
ok! cc @valentinacala

Reply:
I set it as not optional. Anyway, I didn't figure out why, if optional, sometimes it is persisted and sometimes it is not. I tried today uninstalling and reinstalling the app and, while optional, the state is persisted also after rehydration. At the contrary, yesterday, apparently under the same conditions, the same state appeared into the app state at the first PERSIST action done by the app while it wasn't included into the subsequent REHYDRATE payload (disappearing from the app state).